aepp/exportDatasetToCloudStorage.py

```diff
@@ -0,0 +1,345 @@
+#  Copyright 2023 Adobe. All rights reserved.
+#  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License. You may obtain a copy
+#  of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software distributed under
+#  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+#  OF ANY KIND, either express or implied. See the License for the specific language
+#  governing permissions and limitations under the License.
+
+# Internal Library
+
+import aepp
+from aepp import flowservice
+from aepp import destinationinstanceservice
+import os
+import time
+import logging
+from typing import Union
+from aepp.utils import Utils
+from .configs import ConnectObject
+from tenacity import Retrying
+from tenacity import retry_if_result
+from tenacity import wait_fixed
+from tenacity import stop_after_attempt
+
+class ExportDatasetToCloudStorage:
+    """
+    A class for exporting dataset to cloud storage functionality
+    Attributes
+    ----------
+    flow_conn : module
+        flow service connection
+    dis_conn : module
+        destination instance service connection
+    username : str
+        the log in for the os
+    logger: module
+        class logger
+    """
+
+    global flow_conn
```

Comment:
do we need to use global variables, also why are the connection and the flow spec id hardcoded here?

Reply:
removed global, spec ids are constants.